crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -0,0 +1,144 @@
+//! Run a prepass before the main pass to generate depth and/or normals texture, sometimes called a thin g-buffer.
+//! These textures are useful for various screen-space effects and reducing overdraw in the main pass.
+//!
```

Comment:
I would add a smalls section here noting that the prepass only runs for opaque meshes, and the opaque part of alpha masked meshes, and not for transparent meshes.